CONTRIBUTING.md

```diff
@@ -26,29 +26,35 @@ To run tests scripts, you have to install:
 Before creating a contribution, make sure your code passes the following checks
 
 ```shell
-./scripts/test_forge.sh
-./scripts/test_cast.sh
-cargo fmt --check
-cargo lint
+$ cargo test
```

Comment:
I think we should include `scripts/prepare-for-tests.sh` here, otherwise cast tests may fail :(

Reply:
It is mentioned above

Reply:
Ahh yes, sorry didn't check what is above, only saw the changes. You're right, that should suffice.